docs/docs/contribute/website/index.md

```diff
@@ -0,0 +1,28 @@
+---
+comments: true
+---
+
+# Contribute to the Website
+
+This section provides information on how to contribute to the website.
+
+## Location of Files Used to Modify the Website
+
+Here are some key files and directories:
+
+- **`docs/assets/`**: This directory contains all the stylesheets and
+javascript files.
+- **`docs/overrides/`**: This directory contains all the html files.
+- **`website.yml`**: This file contains configurations specific to the
+website, such as all the directories.
+- **`mkdocs.yml`**: The main configuration file for MkDocs.
+It defines the structure and configuration of the documentation site.
+
+## How to Test/Verify Changes
```

Comment:
Thank you for the contribution! Can you please add a section (or link if it already exists) on how to deploy the website locally and check the changes there ?

Reply:
There was a document for that already. I have added the link to it !